src/python/director/treeviewer.py

```diff
@@ -326,26 +320,38 @@ def addColorChannels(polyData, channels):
             colorArray = np.asarray(channels["rgb"]) * 255
             vnp.addNumpyToVtk(polyData, colorArray.astype(np.uint8), "rgb")
 
-    def __init__(self, geomDatas, polyData):
-        self.polyDataItem = vis.PolyDataItem("geometry", polyData, view=None)
-        self.polyDataItem._updateColorByProperty()
+    def __init__(self, geomDatas):
```

Comment:
previously, constructing a Geometry would also construct a new polyDataItem. That's not necessary when there's an existing item at the target path, so now the Geometry just owns a polyData, not a polyDataItem.